File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/UnsubscribeCommand.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class UnsubscribeCommand{

	@SerializedName("commandMetadata")
	private CommandMetadata commandMetadata;

	@SerializedName("clickTrackingParams")
	private String clickTrackingParams;

	@SerializedName("unsubscribeEndpoint")
	private UnsubscribeEndpoint unsubscribeEndpoint;

	public void setCommandMetadata(CommandMetadata commandMetadata){
		this.commandMetadata = commandMetadata;
	}

	public CommandMetadata getCommandMetadata(){
		return commandMetadata;
	}

	public void setClickTrackingParams(String clickTrackingParams){
		this.clickTrackingParams = clickTrackingParams;
	}

	public String getClickTrackingParams(){
		return clickTrackingParams;
	}

	public void setUnsubscribeEndpoint(UnsubscribeEndpoint unsubscribeEndpoint){
		this.unsubscribeEndpoint = unsubscribeEndpoint;
	}

	public UnsubscribeEndpoint getUnsubscribeEndpoint(){
		return unsubscribeEndpoint;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/UnsubscribeCommand.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import com.google.gson.annotations.SerializedName

class UnsubscribeCommand {
    @SerializedName("commandMetadata")
    var commandMetadata: CommandMetadata? = null

    @SerializedName("clickTrackingParams")
    var clickTrackingParams: String? = null

    @SerializedName("unsubscribeEndpoint")
    var unsubscribeEndpoint: UnsubscribeEndpoint? = null
    override fun toString(): String {
        return "UnsubscribeCommand{" +
                "commandMetadata = '" + commandMetadata + '\'' +
                ",clickTrackingParams = '" + clickTrackingParams + '\'' +
                ",unsubscribeEndpoint = '" + unsubscribeEndpoint + '\'' +
                "}"
    }
}